.cirrus.yml

```diff
@@ -103,6 +103,16 @@ build_task:
               # ID for re-use of build output
               CI_DESIRED_RUNTIME: crun
               CI_DESIRED_NETWORK: netavark
+              CI_DESIRED_DATABASE: boltdb
+        - env: &sqliteenvvars
+              DISTRO_NV: ${FEDORA_NAME}
+              # Not used here, is used in other tasks
+              VM_IMAGE_NAME: ${FEDORA_CACHE_IMAGE_NAME}
+              CTR_FQIN: ${FEDORA_CONTAINER_FQIN}
+              # ID for re-use of build output
+              CI_DESIRED_RUNTIME: crun
+              CI_DESIRED_NETWORK: netavark
+              CI_DESIRED_DATABASE: sqlite
```

Comment:
I think this is also going to cause task-name ambiguity since `$CI_DESIRED_DATABASE` is not part of the `&std_name_fmt` alias.  I think adding it to the beginning or end of the name alias will be okay.  Just be beware that github truncates the name with an ellipsis (`...`).  It should still show the full name when clicked on under the "Checks" tab.

Reply:
Correction: I'm wrong, looks like cirrus will add the key/value to the end of the name.  I don't think `hack/get_ci_vm.sh` is that smart though.

Reply:
Well, see, the trouble is, I'm not very human so probably won't give the best advice :grinning:   IMHO, it's important to keep `hack/get_ci_vm.sh` working, so my vote is for updating the `&std_name_fmt` alias.  As soon as you do that, you should be able to run the script and verify it's actually working.

Reply:
I just tried from this PR's code, and `hack/get_ci_vm.sh` doesn't represent the new SQLite tasks `CI_DESIRED_DATABASE` key.  It doesn't break horribly either, so that's good.

I did some experimenting with updating the `std_name_fmt` alias, it only works if a non-empty default var is also added:

```diff
diff --git a/.cirrus.yml b/.cirrus.yml
index 149b7243b..40a3bd19c 100644
--- a/.cirrus.yml
+++ b/.cirrus.yml
@@ -55,6 +55,7 @@ env:
     DISTRO_NV:               # any {PRIOR_,}{FEDORA,DEBIAN}_NAME value
     VM_IMAGE_NAME:           # One of the "Google-cloud VM Images" (above)
     CTR_FQIN:                # One of the "Container FQIN's" (above)
+    CI_DESIRED_DATABASE: boltdb  # One of "", "sqlite", or "boltdb"

     # Curl-command prefix for downloading task artifacts, simply add the
     # the url-encoded task name, artifact name, and path as a suffix.
@@ -563,7 +564,7 @@ windows_smoke_test_task:
 local_integration_test_task: &local_integration_test_task
     # Integration-test task name convention:
     # <int.|sys.> <podman|remote> <Distro NV> <root|rootless>
-    name: &std_name_fmt "$TEST_FLAVOR $PODBIN_NAME $DISTRO_NV $PRIV_NAME $TEST_ENVIRON"
+    name: &std_name_fmt "$TEST_FLAVOR $PODBIN_NAME $DISTRO_NV $PRIV_NAME $TEST_ENVIRON $CI_DESIRED_DATABASE"
     alias: local_integration_test
     # Docs: ./contrib/cirrus/CIModes.md
     only_if: *not_tag_branch_build_docs

```

While I'm not convinced it does much for readability, it does ensure the hack script works in all cases.